examples/sync/src/lib.rs

```diff
@@ -0,0 +1,130 @@
+//! [Any](commonware_storage::adb::any::Any) database sync example library.
+//!
+//! This library demonstrates ADB (Any Database) synchronization between clients and servers.
+//! It includes network protocols, database configuration, and utilities for creating test data.
+//!
+//! The sync example showcases how to:
+//! - Create and configure ADB databases
+//! - Implement network-based resolvers for fetching operations
+//! - Synchronize database state between remote peers
+
+use commonware_cryptography::Hasher as CryptoHasher;
+use commonware_storage::adb::any::Config;
+
+pub mod protocol;
+pub mod resolver;
+
+pub use protocol::*;
+pub use resolver::NetworkResolver;
+
+/// Hasher type used in the database.
+pub type Hasher = commonware_cryptography::sha256::Sha256;
+
+/// Key type used in the database.
+pub type Key = commonware_cryptography::sha256::Digest;
+
+/// Value type used in the database.
+pub type Value = commonware_cryptography::sha256::Digest;
+
+/// Database type alias.
+pub type Database<E> = commonware_storage::adb::any::Any<E, Key, Value, Hasher, Translator>;
+
+/// Operation type alias.
+pub type Operation = commonware_storage::adb::operation::Operation<Key, Value>;
+
+/// Translator type for the database.
+pub type Translator = commonware_storage::translator::EightCap;
+
+/// Create a database configuration with appropriate partitioning.
+pub fn create_adb_config(db_id: &str) -> Config<Translator> {
```

Comment:
I don't think including a `db_id` makes sense (this should be controlled by the outer runtime root file)